src/django/Dockerfile

```diff
@@ -5,8 +5,6 @@ RUN apt-get upgrade -y
 
 # Install GDAL dependencies
 RUN apt-get install -y python3-pip libgdal-dev locales binutils
-RUN ln -s /usr/bin/python3.7 /usr/bin/python
```

Comment:
This seems to be considered dangerous, so we generally avoid doing this within project systems. I do have an alias for this on my development workstation, however. But in project code we try to limit customizations and follow community practice and recommendations as much as possible.

Reply:
Got it - I am not sure where I got the idea that this aliasing was important to have inside of a container. Will get in the habit of explicitly specifying python3 in bash scripting moving forward